research/05-problem-ana.tex

```diff
@@ -48,12 +49,11 @@ \subsubsection{Language specific additional commands}
 should be considered.
 
 One possible solution is to allow for language specific configurations
-that are loaded and used during the generation process of a REPL for
-that language. This can be done by extending the editor services
-discussed in \cref{sec:editor-serv} with REPL commands: similar to
-menu actions, where one can define menu button ``Run'' to run a
-program, one may define a ``load-file'' command and bind it to an
-action.
+that are loaded with the language definition of that language. This
```

Comment:
I'm trying to stay true to what Spoofax actually does.
That is: it loads the language definition in Stratego once when discovering language on a certain path.